plugin/pkg/scheduler/algorithm/priorities/interpod_affinity.go

```diff
@@ -224,7 +224,7 @@ func (ipa *InterPodAffinity) CalculateInterPodAffinityPriority(pod *v1.Pod, node
 	for _, node := range nodes {
 		fScore := float64(0)
 		if (maxCount - minCount) > 0 {
-			fScore = 10 * ((pm.counts[node.Name] - minCount) / (maxCount - minCount))
+			fScore = float64(schedulerapi.MaxPriority) * ((pm.counts[node.Name] - minCount) / (maxCount - minCount))
```

Comment:
optional: The corresponding test files also use literal "10" a lot. It would be great to replace those with `MaxPriority` as well.